oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -71,9 +70,9 @@ struct WasmInterface {
     pretty_name: String,
 
     /// Reader channel mappings to unique u64 handles
-    readers: HashMap<AbiHandle, Handle>,
+    readers: HashMap<oak_abi::Handle, Handle>,
```

Comment:
IIRC you were planning to clean up (remove?) this mapping at some point, right?

Reply:
Yep, incoming once I find a decent block of time to concentrate in.